### PR TITLE
Ported Dockerfile to rockcraft.yaml and started building ROCKs

### PR DESCRIPTION
**Pull request by UtkarshBhatthere**
The new container build system departs from [docker build](https://docs.docker.com/engine/reference/commandline/build/) system to [rockcraft](https://github.com/canonical/rockcraft) which uses a declarative yaml file (rockcraft.yaml) to build OCI compliant container archives. Once the container archive is loaded into docker it can be used directly and is tested similarly to the docker build based container image.

### CODE REVIEW

File: rockcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+summary: Ubuntu based Ceph container image
+description: |
+    Rock for Containerised Ceph based on Ubuntu Ceph distribution. 
+license: MIT
+platforms:
+    amd64:
+
+package-repositories:
+  - type: apt
+    ppa: utkarshbhatthere/ceph-oci
```
Comment (sabaini) [inline rockcraft.yaml:13]: Cosmetic: maybe add a comment that this ppa should be replaced with ubuntu archives once patches land

**Review by sabaini — COMMENTED** [review]
lgtm thanks a lot! One minor doc request for the ppa inline

File: rockcraft.yaml (hunk 1 above)
Comment (UtkarshBhatthere) [reply rockcraft.yaml:15]: Ack

**Comment by sabaini** [comment]
Lgtm \o/

**Pull request merged**